kura/org.eclipse.kura.nm/src/main/java/org/eclipse/kura/nm/ModemManagerDbusWrapper.java

```diff
@@ -49,6 +51,7 @@ public class ModemManagerDbusWrapper {
     private final DBusConnection dbusConnection;
 
     private final Map<String, NMModemResetHandler> modemHandlers = new HashMap<>();
+    private final Map<String, MMFailedModemResetTimer> failedModemHandlers = new HashMap<>();
```

Comment:
These are not "handlers" anymore at this point

```suggestion
    private final Map<String, MMFailedModemResetTimer> failedModemResetTimers = new HashMap<>();
```

Reply:
Argh! :weary: